src/app/replayer/sql.ml

```diff
@@ -119,16 +121,22 @@ module User_command = struct
       Caqti_type.(
         tup4 (tup4 string int int int)
           (tup4 int64 int64 int64 (option int64))
-          (tup2 string int64) (tup4 int int64 int64 int))
+          (tup3 int64 string int64) (tup4 int int64 int64 int))
```

Comment:
could `valid_until` be an `option int64` instead?

Reply:
Yes, that's much cleaner, thanks. Done.